lib/declarative_authorization/test/helpers.rb

```diff
@@ -155,8 +155,14 @@ def this_is_an_abstract_controller_so_it_needs_no_access_tests
         alias :access_tests_not_required :this_is_an_abstract_controller_so_it_needs_no_access_tests
 
         def all_public_actions
-          actions = controller_class.public_instance_methods(false)
-          actions += controller_class.superclass.public_instance_methods(false)
+          actions = []
```

Comment:
Could you update the README as well?
Currently it says:
`Support for Rails 4 and 5`
Instead, I think it should follow the Appraisals:
https://github.com/appfolio/ae_declarative_authorization/blob/master/Appraisals#L1-L2

Reply:
I agree it should be updated, but that seems unrelated to this change.

Reply:
Fair enough.